instruments/tests/test_yokogawa/test_yokogawa_6370.py

```diff
@@ -223,6 +223,67 @@ def test_active_trace():
         assert inst.active_trace == inst.Traces.G
 
 
+# METHODS #
+
+
+@given(values=st.lists(st.decimals(allow_infinity=False, allow_nan=False),
```

Comment:
Great use of hypothesis :+1:

Reply:
Totally stolen from further up in the test suite. But that's one way to start. More to come :smiley: